benchmark/api/query.benchmark.cpp

```diff
@@ -37,7 +38,8 @@ class QueryBenchmark {
     std::shared_ptr<HeadlessDisplay> display{ std::make_shared<HeadlessDisplay>() };
     HeadlessView view{ display, 1 };
     DefaultFileSource fileSource{ "benchmark/fixtures/api/cache.db", "." };
-    Map map{ view, fileSource, MapMode::Still };
+    ThreadPool workerThreadPool{ 4 };
```

Comment:
Rename `workerThreadPool` ⇢ `threadPool` in each of the places in this PR that are gaining a `ThreadPool` member. The fact that it's used for "workers" is an implementation detail.